<launch>
    <node name="gen3_compliant_controllers" pkg="kortex_hardware" type="controller.launch" output="screen">
        <param name="ip_address" value="192.168.1.10" />
        <param name="dof" value="7" />
        <param name="controller_type" value="task" />
    </node>
</launch>

